### PR TITLE
Added a Spanish section

### PR DESCRIPTION
**Pull request by legoman8304**
Literally made a Spanish section of the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,45 @@ yet](https://github.com/GoogleChromeLabs/squoosh/issues/251).
...
+
+* [Datos de visita básicos] (https://support.google.com/analytics/answer/6004245?ref_topic=2919631).
+* Antes y después del tamaño de la imagen una vez que se descarga una imagen. Estos valores se redondean al más cercano.
+  kilobyte.
+
+La compresión de imágenes se maneja localmente; No se envían datos adicionales al servidor.
+
+# Construyendo localmente
+
+Clona el repositorio, y
```
Comment (JuanCrg90) [inline README.md]: what do you think about change this to

> Clona el repositorio, y ejecuta:

File: README.md [hunk 2]
```diff
@@ -27,7 +27,45 @@ yet](https://github.com/GoogleChromeLabs/squoosh/issues/251).
...
+
+```
+npm instalar
```
Comment (JuanCrg90) [inline README.md]: what do you think about to keep this in english, this won't run in a terminal 🤔 

```
npm install
```

File: README.md [hunk 3]
```diff
@@ -27,7 +27,45 @@ yet](https://github.com/GoogleChromeLabs/squoosh/issues/251).
...
+npm ejecutar construir
```
Comment (JuanCrg90) [inline README.md]: what do you think about to keep this in english, this won't run in a terminal 🤔 
```
npm run build
```

**Comment by surma** [comment]
@legoman8304 @JuanCrg90 Thanks for the effort to translate the README! We really appreciate the work you put into this. That being said, none of us in the team speak Spanish to be able to maintain the Spanish version of the README. We expect the project to evolve in the future so we would like to avoid having a stale (and — for us — unfixable) README in the repository.

I’ll go ahead and close this.

**Comment by kosamari** [comment]
Hi @legoman8304 first of all, thanks for the contribution! 
Multi-language documentation/resources is something I care a lot. Unfortunetly sqoosh project is not set up to have them in our repo (yet, maybe someday!)

You are more than welcome to write blog/put it in gist of your Spanish translation. Please let us know if you do & we'll share with others :)

**Pull request closed without merging**